WeatherToday/Sources/Data/Repositories/WeatherRepository.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  WeatherRepository.swift
+//  WeatherToday
+//
+//  Created by Taher on 7/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+final class WeatherRepository: WeatherRepositoryProtocol {
+
+    private var networkApi: NetworkAPIProtocol
+
+    init(network: WeatherNetworkAPI) {
+        networkApi = network
+    }
+
+    func getWeatherData(cityName: String) -> Single<WeatherApi> {
+        let weatherApiConfiguration = WeatherApiRequestConfigurations.get(cityName: cityName)
+        return networkApi.performRequest(
+            weatherApiConfiguration,
+            for: WeatherApi.self
+        ).map { $0 as WeatherApi }
```

Comment:
Please check line break for this as well.
```suggestion
        return networkApi
        .performRequest(
            weatherApiConfiguration,
            for: WeatherApi.self
        )
        .map { $0 as WeatherApi }
```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/0df0b8e6ef1945dbb46c9bd130e52d369b861265)